frontend/lib/requests/authorized-user.ts

```
@@ -316,15 +324,30 @@ export async function fetchWebsiteCreators(): Promise<AuthorizedUser[]> {
+
     const response = await fetch(
       NEXT_PUBLIC_STRAPI_API_URL + "/api/authorized-users?" + query,
       {
         headers: { Authorization: "Bearer " + STRAPI_ACCESS_TOKEN },
         cache: "no-store",
       },
     );
+
     const data = await response.json();
-    return flattenAttributes(data.data);
+    let creators: AuthorizedUser[] = flattenAttributes(data.data);
```

ESLint

[error] 'creators' is never reassigned. Use 'const' instead.

(prefer-const)